packages/amplify-cli-core/src/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from './open';
```

Comment:
nit can get rid of this file and change the export in the main index.ts to export directly from open.ts

Reply:
I'd keep the single level export for clarity as we probably not exporting everything in an upper level directory.